config/application.rb

```diff
@@ -4,11 +4,22 @@
 
 require "rails/all"
 
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module LagoApi
   class Application < Rails::Application
-    config.load_defaults(7.1)
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.2
+
+    # Disable YJIT as we are not ready yet
+    config.yjit = false
+
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w[tasks])
```

Comment:
I think we'll be able to remove the explicit require (like with the lago http client)!

Reply:
I have removed it. thanks.